pandas/core/window/ewm.py

```diff
@@ -252,7 +253,9 @@ def __init__(
                 raise ValueError(
                     "halflife must be a string or datetime.timedelta object"
                 )
-            self.times = np.asarray(times.astype(np.int64))
+            if isna(times).any():
+                raise ValueError("Cannot convert NaT values to integer")
```

Comment:
Could you add a test for this case?

Reply:
double-checked and you're right we dont have any.  suggestions for what this might look like?

Reply:
Something simple like

```
with pytest.raises(ValueError, match=...):
     Series(range(1)).ewm(com=0.1, times=to_datetime(['NaT'])))
```

works.